content/en/docs/concepts/scheduling/customization-and-extension.md

```diff
@@ -0,0 +1,308 @@
+- `PodFitsResources`: Checks if the free resource (CPU and Memory) meets the requirement of the Pod. The free resource is measured by the capacity minus the sum of requests of all Pods on the node. To learn more about the resource QoS in Kubernetes, please check QoS proposal.
+- `PodMatchNodeSelector`:Checks if a pod node selector matches the node label.
+- `NoVolumeZoneConflict`: Evaluate if the volumes a pod requests are available on the node, given the Zone restrictions.
+- `MaxEBSVolumeCount`: Ensures that the number of attached ElasticBlockStore volumes does not exceed a maximum value (by default, 39, since Amazon recommends a maximum of 40 with one of those 40 reserved for the root volume -- see Amazon's documentation). The maximum value can be controlled by setting the KUBE_MAX_PD_VOLS environment variable.
+- `MaxGCEPDVolumeCount`: Ensures that the number of attached GCE PersistentDisk volumes does not exceed a maximum value (by default, 16, which is the maximum GCE allows -- see GCE's documentation). The maximum value can be controlled by setting the KUBE_MAX_PD_VOLS environment variable.
+- `MaxCSIVolumeCount`:  Decides how many CSI volumes should be attached.
+- `MaxAzureDiskVolumeCount`: Fit is determined by whether or not there would be too many Azure Disk volumes attached to the node.
+- `NoDiskConflict`: Evaluates if a pod can fit due to the volumes it requests, and those that are already mounted. If there is already a volume mounted on that node, another pod that uses the same volume can't be scheduled there. This is GCE, Amazon EBS, and Ceph RBD specific for now:
+  - GCE PD allows multiple mounts as long as they're all read-only
+  - AWS EBS forbids any two pods mounting the same volume ID
+  - Ceph RBD forbids if any two pods share at least same monitor, and match pool and image.
+  - ISCSI forbids if any two pods share at least same IQN, LUN and Target
+- `GeneralPred`: Checks whether noncriticalPredicates and EssentialPredicates pass. noncriticalPredicates are the predicates that only non-critical pods need and EssentialPredicates are the predicates that all pods, including critical pods, need.
+- `CheckNodeMemoryPressure`: Checks if a pod can be scheduled on a node reporting memory pressure condition.
+- `CheckNodePIDPressure`: Checks if a pod can be scheduled on a node reporting pid pressure condition.
+- `CheckNodeDiskPressure`: Checks if a pod can be scheduled on a node reporting disk pressure condition.
+- `CheckNodeCondition`: Checks if a pod can be scheduled on a node reporting out of disk, network unavailable and not ready condition. Only node conditions are accounted in this predicate.
+- `PodToleratesNodeTaints`: checks if a pod tolerations can tolerate the node taints.
+- `CheckVolumeBinding`: Evaluates if a pod can fit due to the volumes it requests, for both bound and unbound PVCs.
+- CheckNodeUnschedulablePred: Checks if a pod can be scheduled on a node with Unschedulable spec.
```

Comment:
Please use the same formatting for the predicate names, to be consistent with the above.